src/Tizen.NUI.Components/Controls/Scrollbar.cs

```diff
@@ -457,18 +457,6 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override void ApplyStyle(ViewStyle viewStyle)
         {
-            if (viewStyle is ScrollbarStyle scrollbarStyle)
```

Comment:
could you check the warning above at line 438?
null check of `size` seems to be required.

Reply:
Done. Thank you!